Header header

float32 right_knee_fix_link_angle
float32 left_knee_fix_link_angle
int32   right_number_revolutions
int32   left_number_revolutions
int32   right_limit
int32   left_limit
